### DISCUSSION TITLE
Go implementation of Filesystem Server

### DISCUSSION THREAD

**Discussion by ezynda3** [modelcontextprotocol/.github#46] [Show and tell]
### Pre-submission Checklist

- [X] This post follows the MCP community guidelines

### What would you like to share?

I've gone ahead and "dogfooded" my Go MCP library and build Go version of the filesystem server. If there is any who has go installed on their system and would like to test it out in Claude I would appreciate the feedback. Instructions are in the repo.

Cheers!

### Relevant Links

https://github.com/mark3labs/mcp-filesystem-server

**Comment by dsp-ant** [comment]
Very cool! I love to see it! I'll give this a spin. I am quite excited about having a Go SDK for MCP, since the Go deployment story is just much easier than Node and Python.
